pkg/runner/client.go

```diff
@@ -173,13 +172,6 @@ func CreateClient(r prometheus.Registerer, cfg *Config) (*pgclient.Client, error
 		err = cfg.DatasetCfg.Apply(context.Background(), conn)
 	} else if cfg.DatasetConfig != "" {
 		err = applyDatasetConfig(context.Background(), conn, cfg.DatasetConfig)
-	} else {
-		// We apply downsampling settings even when DatasetConfig is not given, which is the most common case.
-		rollupCfg := &rollup.Config{}
-		err = rollupCfg.Apply(context.Background(), conn)
-		if err != nil {
-			return nil, fmt.Errorf("error applying downsampling configuration: %w", err)
-		}
```

Comment:
Please ignore this. This was the result of a bad conflict resolution. We do not plan to apply downsampling if dataset-config and downsampling is explicitly given, in order to be in the safe place.